### PR TITLE
Added styling

### PR DESCRIPTION
**Pull request by sebalp100**
# Added styling

For the last milestone, we added the following:

- General style using the next template: https://pedantic-beaver-cb923b.netlify.app/#/
- Finalized the website
- React icons

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
 
 function Header() {
   return (
     <nav>
       <h1>Bookstore CMS</h1>
       <div className="link">
-        <li><Link to="/" style={{ textDecoration: 'none' }}>Books</Link></li>
-        <li><Link to="/Categories" style={{ textDecoration: 'none' }}>Categories</Link></li>
+        <li><Link to="/" style={{ textDecoration: 'none' }}>BOOKS</Link></li>
+        <li><Link to="/Categories" style={{ textDecoration: 'none' }}>CATEGORIES</Link></li>
```
Comment (mukhammadrizooff) [inline src/components/Header.js]: - [x] [REQUIRED] Good job so far! Please implement correct colors and use color effects(`Pressed` and `Enebled`) like below:

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @sebalp100, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sebalp100** [comment]
Changes required implemented.

**Review by t-yanick — APPROVED** [review]
Hi @sebalp100, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**